"The Checklist"
===============

http://www.medschool.umaryland.edu/Genetic_Variant_Interpretation_Tool1.html/

The "most important" criteria on the checklist being: 
* PM1 Located in a mutational hot spot and/or critical and well-established functional domain (e.g., active site of an enzyme) without benign variation
* PM2 Absent from controls (or at extremely low frequency if recessive) in Exome Sequencing Project, 1000 Genomes Project, or Exome Aggregation Consortium
* PVS1 null variant (nonsense, frameshift, canonical Â±1 or 2 splice sites, initiation codon, single or multiexon deletion) in a gene where LOF is a known mechanism of disease. 
List of fielsd for  "Variant_Classification" colunm https://gatkforums.broadinstitute.org/gatk/discussion/8815/oncotator-variant-classification-and-secondary-variant-classification
